### ISSUE TITLE
States JSON parsing with ArduinoJSON issue

### ISSUE THREAD

**Issue by abferguson** [COVID19Tracking/issues#294]
Up until sometime Thursday, April 23, I was able to use ArduinoJSON 6.15.1 to parse the states JSON:
  https://covidtracking.com/api/v1/states/current.json

Currently, my MCU crashes when I try to parse the JSON object.  Interestingly, I have no trouble parsing _https://covidtracking.com/api/v1/us/current.json_ (US stats)

Did something change in the states data in the last day and a half?

**Comment by abferguson** [comment]
Coding workaround.

**Comment by muamichali** [comment]
Hi @abferguson 

The current JSON passed validation at jsonlint.com 

Do you get any specific error when parsing it? I know that the value of the notes field changed around that time.

**Comment by abferguson** [comment]
There is NO issue with the data from the website. Being my first time using ArduinoJSON with a much larger HTTP GET data set than previous experience, I just needed to learn and apply new techniques to solve the issue.

My issue was that the states JSON data just got too big for my MCU memory capacity. The unit would just spontaneously reboot when trying to deserialize the states data with a JSON capacity to fit the data. 

I went back to the ArduinoJSON documentation to get smarter and discovered the filtering capability. This feature reduces memory usage significantly, in my case, thus avoiding crashes. 

The filtering is a Very cool capability.  Thanks ArduinoJSON authors.
